fe/src/components/Sidebar/Drawerbar/DrawerbarContent.tsx

```diff
@@ -0,0 +1,21 @@
+import { Flex } from '@chakra-ui/react';
+// eslint-disable-next-line
+import { IoHome } from 'react-icons/io5';
+import React from 'react';
+import SidebarItem from '../SidebarElements/SidebarItem';
+import SidebarItemLink from '../SidebarElements/SidebarItemLink';
```

Comment:
Hmm yes, i see that eslint is not happy about such imports.
```
/home/travis/build/TPT-Loane/TPT-Loane/fe/src/components/Header/Header.tsx
  10:27  error  Reaching to "../../utils/nav" is not allowed  import/no-internal-modules
/home/travis/build/TPT-Loane/TPT-Loane/fe/src/components/Sidebar/Drawerbar/DrawerbarContent.tsx
  5:25  error  Reaching to "../SidebarElements/SidebarItem" is not allowed      import/no-internal-modules
  6:29  error  Reaching to "../SidebarElements/SidebarItemLink" is not allowed  import/no-internal-modules
  7:27  error  Reaching to "../../../utils/nav" is not allowed                  import/no-internal-modules
/home/travis/build/TPT-Loane/TPT-Loane/fe/src/components/Sidebar/Sidebar.tsx
  3:32  error  Reaching to "./SidebarElements/SidebarContent" is not allowed  import/no-internal-modules
```

Two points i recommend or something similar to them.

**1)** Allow import from `util ` folder for now (or until we find a final placement for single source of truth things such as this): update .eslintrc
```
["error", { "allow": ["**/assets/*", "**/styles/*", "**/index.ts", "**/utils/*"] }]
```

**2)** Rearrange sidebar components.

Currently you see eslint is not happy because you are importing **from SidebarElements** folder **to Drawerbar/DrawerbarContent**.

Same for **Sidebar**, you are importing from another folder structure **SidebarElements**.

I'm thinking about using some folder structure similar to this: 🤔 
```
components/Sidebar/index,ts
components/Sidebar/Sidebar.tsx
components/Sidebar/SidebarContent.tsx
components/Sidebar/SidebarItem.tsx
components/Sidebar/SidebarItemButton.tsx
components/Sidebar/SidebarItemLink.tsx
components/Drawerbar/index.ts
components/Drawerbar/Drawerbar.tsx
components/Drawerbar/DrawerbarContent.tsx
```

And by one solution you could make some components from `Sidebar `folder "public", by adding them to `index.ts`, because currently as i understand `DrawerbarContent ` is using `SidebarItem `and `SidebarItemLink ` components from `Sidebar `component.

This means that there would be multiple imports from `Sidebar`.

Or maybe someone has better ideas? 🤔

Reply:
> Allow import from util folder for now (or until we find a final placement for single source of truth things such as this): update .eslintrc

Why update the lint configs? Just export the public stuff from the utils folder out the same way we do components (using `index.ts`) and all will be swell


I like the proposed file structure btw

Reply:
> Why update the lint configs? Just export the public stuff from the utils folder out the same way we do components (using `index.ts`) and all will be swell


Yep, after seeing your comment, i agree on that. 👍